### PR TITLE
http only flag

### PR DESCRIPTION
**Pull request by TechAlchemistry**

### CODE REVIEW

**Comment by TechAlchemistry** [comment]
I think, my previously PR can be closed, because this PR include all my commits. In the future, I will try to avoid it.

**Comment by TechAlchemistry** [comment]
Only one test is failed. 

1) Serps\Test\HttpClient\CurlClientTest::testProxy
Failed asserting that 404 matches expected 200.

It's because that proxy is not working?

File: README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ SERPS - CURL Client
 
 This is a CURL Client implementation for [SERPS](https://github.com/serp-spider/serps)
 
+Requirements
+-------
+cURL >= 7.48.0
+
```
Comment (gsouf) [inline README.md]: This is not desired here, curl information are placed in the documentation repository

Comment (gsouf) [inline README.md]: You should update this file instead https://github.com/serp-spider/documentation/blob/master/docs/http-client/curl.md

**Comment by gsouf** [comment]
I added a side note about the readme file, can  you remove this change and I will merge.

For the proxy, I know, http tests are still experimental and need to be updated

**Comment by TechAlchemistry** [comment]
I removed changes of readme file and create [issue in serp-spider/documentation](https://github.com/serp-spider/documentation/issues/6)

**Pull request merged**